src/hammer-tech/hammer_tech.py

```diff
@@ -404,6 +404,20 @@ def dont_use_list(self) -> Optional[List[str]]:
             dont_use_list = list(map(lambda x: str(x), list(dont_use_list_raw)))
             return dont_use_list
 
+    @property
+    def physical_only_cells_list(self) -> Optional[List[str]]:
+        """
+        Get the list of physical only  cells.
+        :return: List of physical only cells, or None if the technology does not define such a list.
+        """
+        physical_only_cells_list_raw = self.config.physical_only_cells_list  # type: Optional[List[str]]
+        if physical_only_cells_list_raw is None:
+            return None
+        else:
+            # Work around the weird objects implemented by the jsonschema generator.
+            physical_only_cells_list = list(map(lambda x: str(x), list(physical_only_cells_list_raw)))
```

Comment:
Getting into the holy war a bit:
```suggestion
            physical_only_cells_list = [str(x) for x in physical_only_cells_list_raw]
```

Reply:
Just FYI that `map(func, ...)` is faster than its list comprehension equivalent as opposed to `map(lambda x: <custom logic>, ...)` so would be good to favour `map(str, ...)` in this case